### ISSUE TITLE
auto-render options not documented

### ISSUE THREAD

**Issue by edemaine** [KaTeX/KaTeX#1944]
I was surprised to not see documentation of the special options supported by `renderMathInElement`, in particular `delimiters` which we often get questions about, in [the auto-render extension documentation page](https://katex.org/docs/autorender.html). Before I forget, we should add that, ideally along with a common alternative for `delimiters` (and warnings about order).

**Comment by jason-s** [comment]
Huh, that's timely, I just ran into the same issue. Would someone mind posting an example here? Because I want to configure the delimiters and I don't know how.

**Comment by jason-s** [comment]
...so a complete workable example seems to be:

```
<script type="text/javascript">
  document.addEventListener("DOMContentLoaded", function() {
    renderMathInElement(document.body, {
      delimiters: [
        {left: "$$", right: "$$", display: true},
        {left: "\\(", right: "\\)", display: false},
      ]
    });
  });
</script>
```

**Comment by kevinbarabash** [comment]
@jason-s would be interested in updating the example in https://github.com/KaTeX/KaTeX/blob/master/docs/autorender.md?

**Comment by kevinbarabash** [comment]
@edemaine https://github.com/KaTeX/KaTeX/blob/master/docs/autorender.md does mention `delimiters` but agree that it needs a warning about the order.  Are there options that are missing from that page?

**Comment by edemaine** [comment]
@kevinbarabash Ah, indeed it does, but that's not getting output to https://katex.org/docs/autorender.html . Docusaurus bug?

**Comment by kevinbarabash** [comment]
Thanks for pointing that out.  Sounds like it could be a Docusaurus bug.

**Comment by kevinbarabash** [comment]
It turns out it's not a Docusaurus bug.  There's an unclosed HTML comment in that .md file which is causing the issue.
<img width="969" alt="Screen Shot 2019-05-13 at 9 50 31 AM" src="https://user-images.githubusercontent.com/1044413/57626875-cf809280-7564-11e9-82a5-19d4308d8f5a.png">

**Comment by ronkok** [comment]
This issue is resolved by PR #1979. After the next KaTeX update, the documentation at https://katex.org/docs/autorender.html will be complete. Until then, the complete version is visible at https://katex.org/docs/next/autorender.html.
